### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by SiansandaTamara**
Add index.html
Add styles.css
Add touch button 
Add form inputs fields

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,13 +169,32 @@ <h3 class="lang-text">CSS</h3>
...
           </div>
         </div>
     </section>
+    <section class="contact-section">
+      <div class="contact-form">
+        <div class="contact-heading">
+          <h2 class="con-title">Contact me</h2>
+          <p class="con-paragraph">
+            If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding.
```
Comment (Donard97) [inline index.html:185]: - [ ] You did an amazing job on this milestone but please move the text more to the left following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Keep coding 💪 

Here you have a screenshot from your project:

image

Here you have a screenshot from the design:

image

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hi @SiansandaTamara  , 👋  

_You did a tremendous job so far! 💯 👏_ 

_**There are some issues that you still need to work on to go to the next project but you are almost there! 💪**_ 

hgiphy


## Highlights ✅ 🏁 

✔️  There is no linter error 
✔️  Gitflow is used correctly
✔️  The Readme file is professional


## Required Changes ♻️

- [ ] Please check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by SiansandaTamara** [comment]
Hi there,I'm have difficulties to change my project what should  I do.it can't moveSent from my Samsung Galaxy smartphone.
-------- Original message --------From: Donard Golaj ***@***.***> Date: 24/03/2022  14:12  (GMT+02:00) To: SiansandaTamara/Portfolio ***@***.***> Cc: tamara siansanda ***@***.***>, Mention ***@***.***> Subject: Re: [SiansandaTamara/Portfolio] Contact form (PR #3) 

@Donard97 requested changes on this pull request.



Hi @SiansandaTamara  , 👋

You did a tremendous job so far! 💯 👏

There are some issues that you still need to work on to go to the next project but you are almost there! 💪



Highlights ✅ 🏁

✔️  There is no linter error

✔️  Gitflow is used correctly

✔️  The Readme file is professional

Required Changes ♻️



 Please check the comments under the review.



Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.



As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.





In index.html:

> +          <h2 class="con-title">Contact me</h2>

+          <p class="con-paragraph">

+            If you have an application you are interested in developing,

+            a feature that you need built or a project that needs coding.





 You did an amazing job on this milestone but please move the text more to the left following Figma design. Keep coding 💪



Here you have a screenshot from your project:



Here you have a screenshot from the design:





—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

[

{

***@***.***": "http://schema.org",

***@***.***": "EmailMessage",

"potentialAction": {

***@***.***": "ViewAction",

"target": "https://github.com/SiansandaTamara/Portfolio/pull/3#pullrequestreview-920166378",

"url": "https://github.com/SiansandaTamara/Portfolio/pull/3#pullrequestreview-920166378",

"name": "View Pull Request"

},

"description": "View this Pull Request on GitHub",

"publisher": {

***@***.***": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]

**Comment by Donard97** [comment]
Hello @SiansandaTamara, I hope you are well. Please reach out to me on Slack.

**Comment by SiansandaTamara** [comment]
## The following were implemented in this project.
- Add footer
- Add flex direction
- Add align item
- Add width
- button  action

**Review by keroloussamy — APPROVED** [review]
Hi @SiansandaTamara, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**